ocaml/xapi-cli-server/record_util.ml

```diff
@@ -1226,3 +1226,16 @@ let update_sync_frequency_of_string s =
       `weekly
   | _ ->
       raise (Record_failure ("Expected 'daily', 'weekly', got " ^ s))
+
+let origin_to_string = function `remote -> "remote" | `bundle -> "bundle"
```

Comment:
The name seems too general in this level. Could it be `repo_origin_to_string`? Or it could be wrapped in a module.